flow_action_components/CollectionProcessors/force-app/main/default/classes/SortCollection.cls

```diff
@@ -97,13 +97,89 @@ global with sharing class SortCollection {
                         returnValue = -1 * direction;
                     }
                 }
+             } else if (fieldType == 'BOOLEAN') {
+                Boolean origValue = Boolean.valueOf(acc.get(this.field));
+                Boolean compareToValue = Boolean.valueOf(compareToAcc.acc.get(this.field));
+                if (origValue == null && compareToValue == null) {
+                    returnValue = 0;
+                } else if (origValue == null) {
+                    returnValue = -1;
+                } else if (compareToValue == null) {
+                    returnValue = 1;
+                } else {
+                    if (origValue && !compareToValue) {
+                        returnValue = 1 * direction;
+                    } else if (!origValue && compareToValue) {
+                        returnValue = -1 * direction;
+                    }
+                }
+             } else if (fieldType == 'CURRENCY') {
+                Decimal origValue = Decimal.valueOf(acc.get(this.field)+'');
+                Decimal compareToValue = Decimal.valueOf(compareToAcc.acc.get(this.field)+'');
+                if (origValue == null && compareToValue == null) {
```

Comment:
It looks like there's an opportunity to DRY this if statement up, as it appears to be repeated without variation. Could you put it into a separate function and have all fo the field type sections call that function?

Reply:
Yes. I'll see what I can do!

Reply:
I've added support for other field types and have been running Apex unit tests. I need some help though: When creating custom fields, which field types corresponding to DECIMAL and INTEGER?
E.g., NUMBER(10,8) corresponds to DOUBLE.
Thanks!

Reply:
The code has been updated and tested. Do I submit another pull request?
Testing via SortCollectionTest.cls and spot checking using the FlowBuilder.